spec/queries/keywords_query_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe(KeywordsQuery) do
+  describe '#call' do
+    context 'given NO keywords' do
+      it 'returns an empty list' do
+        keywords = described_class.new.call
+
+        expect(keywords).to(be_empty)
+      end
+    end
+
+    context 'given 5 keywords' do
```

Comment:
Such test can be misleading as it should work for any number of keywords:

```suggestion
    context 'given a list of keywords' do
```

Reply:
Fixed in e5f09346cd54e1b94fc42bbaa94df8637871bd91